src/components/local-scene-sidebar/local-priority-card/local-priority-card-styles.module.scss

```diff
@@ -0,0 +1,38 @@
+@import 'styles/settings';
+@import 'styles/typography-extends';
+
+.title {
+  @extend %_temporaryTitle;
+  margin: 0;
+  color: $_temporary-dark-text;
+  text-align: center;
+}
+
+.datasetWrapper {
+  display: flex;
+}
+
+.wdpaIcon,
+.priorityIcon {
+  min-width: 25px;
+  margin-right: 10px;
+}
+
+.wdpaIcon {
+  fill: #FF6C47;
+}
+
+.priorityIcon {
+  fill: #FFD413;
```

Comment:
Can we add `#FFD413` color to `settings.scss` file? Or use the similar one that we already have?